From 46d04bf3e83fbd366cada1fd185a492c8fc80e87 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 13 Mar 2012 17:00:06 +0800
Subject: ENGR00176649-2 sabrelite_SD ARD pfuze: update board level for
 pfuze100 1.0

modify SW1xSTANDBY value,Update for Sabrelite_SD board and Sabrelite_ARD board

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c |   11 +++--------
 arch/arm/mach-mx6/mx6q_sabresd_pmic_pfuze100.c   |   11 +++--------
 2 files changed, 6 insertions(+), 16 deletions(-)

diff --git a/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c b/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
index 6fcefd6..cc12a02 100644
--- a/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
+++ b/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
@@ -41,13 +41,13 @@
 
  /*SWBST*/
 #define PFUZE100_SW1ASTANDBY    33
-#define PFUZE100_SW1ASTANDBY_STBY_VAL   (0x14)
+#define PFUZE100_SW1ASTANDBY_STBY_VAL   (0x18)
 #define PFUZE100_SW1ASTANDBY_STBY_M     (0x3f<<0)
 #define PFUZE100_SW1BSTANDBY   40
-#define PFUZE100_SW1BSTANDBY_STBY_VAL  (0x14)
+#define PFUZE100_SW1BSTANDBY_STBY_VAL  (0x18)
 #define PFUZE100_SW1BSTANDBY_STBY_M    (0x3f<<0)
 #define PFUZE100_SW1CSTANDBY    47
-#define PFUZE100_SW1CSTANDBY_STBY_VAL   (0x14)
+#define PFUZE100_SW1CSTANDBY_STBY_VAL   (0x18)
 #define PFUZE100_SW1CSTANDBY_STBY_M     (0x3f<<0)
 #define PFUZE100_SW2STANDBY     54
 #define PFUZE100_SW2STANDBY_STBY_VAL    0x0
@@ -392,11 +392,6 @@ static struct regulator_init_data vgen6_init = {
 static int pfuze100_init(struct mc_pfuze *pfuze)
 {
 	int ret;
-	ret = pfuze_reg_rmw(pfuze, PFUZE100_SWBSTCON1,
-			    PFUZE100_SWBSTCON1_SWBSTMOD_M,
-			    PFUZE100_SWBSTCON1_SWBSTMOD_VAL);
-	if (ret)
-		goto err;
 	ret = pfuze_reg_rmw(pfuze, PFUZE100_SW1ASTANDBY,
 			    PFUZE100_SW1ASTANDBY_STBY_M,
 			    PFUZE100_SW1ASTANDBY_STBY_VAL);
diff --git a/arch/arm/mach-mx6/mx6q_sabresd_pmic_pfuze100.c b/arch/arm/mach-mx6/mx6q_sabresd_pmic_pfuze100.c
index ab94b11..f6a313e 100644
--- a/arch/arm/mach-mx6/mx6q_sabresd_pmic_pfuze100.c
+++ b/arch/arm/mach-mx6/mx6q_sabresd_pmic_pfuze100.c
@@ -40,13 +40,13 @@
 #define PFUZE100_I2C_ADDR         (0x08)
  /*SWBST*/
 #define PFUZE100_SW1ASTANDBY	33
-#define PFUZE100_SW1ASTANDBY_STBY_VAL	(0x14)
+#define PFUZE100_SW1ASTANDBY_STBY_VAL	(0x18)
 #define PFUZE100_SW1ASTANDBY_STBY_M	(0x3f<<0)
 #define PFUZE100_SW1BSTANDBY   40
-#define PFUZE100_SW1BSTANDBY_STBY_VAL  (0x14)
+#define PFUZE100_SW1BSTANDBY_STBY_VAL  (0x18)
 #define PFUZE100_SW1BSTANDBY_STBY_M    (0x3f<<0)
 #define PFUZE100_SW1CSTANDBY	47
-#define PFUZE100_SW1CSTANDBY_STBY_VAL	(0x14)
+#define PFUZE100_SW1CSTANDBY_STBY_VAL	(0x18)
 #define PFUZE100_SW1CSTANDBY_STBY_M	(0x3f<<0)
 #define PFUZE100_SW2STANDBY     54
 #define PFUZE100_SW2STANDBY_STBY_VAL    0x0
@@ -345,11 +345,6 @@ static struct regulator_init_data vgen6_init = {
 static int pfuze100_init(struct mc_pfuze *pfuze)
 {
 	int ret;
-	ret = pfuze_reg_rmw(pfuze, PFUZE100_SWBSTCON1,
-			    PFUZE100_SWBSTCON1_SWBSTMOD_M,
-			    PFUZE100_SWBSTCON1_SWBSTMOD_VAL);
-	if (ret)
-		goto err;
 	ret = pfuze_reg_rmw(pfuze, PFUZE100_SW1ASTANDBY,
 			    PFUZE100_SW1ASTANDBY_STBY_M,
 			    PFUZE100_SW1ASTANDBY_STBY_VAL);
-- 
1.7.9.5

